Projects/Cyber-Range-K8/nginx_modify_content.md

```diff
@@ -0,0 +1,93 @@
+# Updating the Contents of an Nginx Webpage in Kubernetes
+
+## TOC
+- [Updating the Contents of an Nginx Webpage in Kubernetes](#updating-the-contents-of-an-nginx-webpage-in-kubernetes)
```

Comment:
The TOC can be updated to remove the document title and TOC entry.

<!- omit-from-toc --> is something that can be used if you generate this with a markdown tool. (Look up your specific tool)